envbuilder.go

```diff
@@ -169,6 +169,7 @@ func run(ctx context.Context, opts options.Options, execArgs *execArgsInfo) erro
 		RemoteEnv:    make(map[string]string),
 	}
 	if fileExists(opts.Filesystem, workingDir.Image()) {
+		opts.Logger(log.LevelInfo, "Found magic image file at %s", workingDir.Image())
```

Comment:
I'd have gone with debug, but I think your choice of info is better. This'll be helpful in understanding what type of image a user is running.

Reply:
Yeah it's way too easy to miss this otherwise!